tests/test_basic_functionality.py

```diff
@@ -209,6 +209,22 @@ def test_connect_via_netcat(run_server, helpers):
             pass
 
 
+def test_on_registration_messages(run_server, helpers):
+    nc = socket.socket()
+    nc.connect(("localhost", 6667))
+    nc.sendall(b"NICK nc\n")
+    nc.sendall(b"USER nc 0 * :netcat\n")
+
+    assert helpers.receive_line(nc) == b":mantatail 001 nc :Welcome to Mantatail nc!nc@127.0.0.1\r\n"
+    assert b":mantatail 002 nc :Your host is Mantatail[" in helpers.receive_line(nc)
+    assert b":mantatail 003 nc :This server was created" in helpers.receive_line(nc)
+    assert b":mantatail 004 nc Mantatail 0.0.1" in helpers.receive_line(nc)
+    assert b":mantatail 005 nc" and b":are supported by this server" in helpers.receive_line(nc)
```

Comment:
This doesn't do what you intended. `"foo" and "bar" in baz` means `"foo" and ("bar" in baz)`, so it ends up doing `and` between a string and a boolean.

```
>>> b":mantatail 005 nc" and True
True
>>> b":mantatail 005 nc" and False
False
```

```suggestion
    line = helpers.receive_line(nc)
    assert b":mantatail 005 nc" in line
    assert b":are supported by this server" in line
```